.github/workflows/release.yaml

```
@@ -6,7 +6,7 @@ on:
       - "*/v*"
 
 jobs:
-  package_name:
+  package-name:
```

Let's not change this b/c we don't need to (I see at one place where it got missed in the yaml).

```suggestion
  package_name:
```